website/data/community.json

```diff
@@ -40,14 +44,7 @@
       "infoLink": "https://github.com/finos/fdc3-desktop-agent",
       "docsLink": "https://github.com/finos/fdc3-desktop-agent",
       "type": "platform-provider",
-      "badges": [
-        {
-          "text": "FDC3 1.2 Supported"
-        },
-        {
-          "text": "Open Source"
-        }
-      ],
+      "badges": [],
```

Comment:
Not keeping the open source tag?

Reply:
no - losing badges for desktop agents, it's just too cluttered